lib/livebook_web/live/output/input_component.ex

```diff
@@ -171,11 +171,70 @@ defmodule LivebookWeb.Output.InputComponent do
     """
   end
 
+  defp input_output(%{attrs: %{type: :datetime}} = assigns) do
+    ~H"""
+    <input
+      id={@id}
+      type="datetime-local"
+      data-el-input
+      class="input w-auto invalid:input--error"
+      name="html_value"
+      value={@value && Calendar.strftime(@value, "%Y-%m-%dT%H:%M")}
+      phx-hook="UtcDateTimeInput"
+      phx-debounce="blur"
+      phx-target={@myself}
+      min={@attrs.min && Calendar.strftime(@attrs.min, "%Y-%m-%dT%H:%M")}
+      max={@attrs.max && Calendar.strftime(@attrs.max, "%Y-%m-%dT%H:%M")}
+      step={@attrs.step}
+      autocomplete="off"
+    />
+    """
+  end
+
+  defp input_output(%{attrs: %{type: :time}} = assigns) do
+    ~H"""
+    <input
+      id={@id}
+      type="time"
+      data-el-input
+      class="input w-auto invalid:input--error"
+      name="html_value"
+      value={@value}
+      phx-hook="UtcDateTimeInput"
+      phx-debounce="blur"
+      phx-target={@myself}
+      min={@attrs.min}
+      max={@attrs.max}
+      step={@attrs.step}
+      autocomplete="off"
+    />
+    """
+  end
+
+  defp input_output(%{attrs: %{type: type}} = assigns)
+       when type in [:date, :week, :month] do
+    ~H"""
+    <input
+      type={html_input_type(type)}
+      data-el-input
+      class="input w-auto invalid:input--error"
+      name="html_value"
+      value={@value}
+      phx-debounce="blur"
+      phx-target={@myself}
+      min={@attrs.min}
+      max={@attrs.max}
+      step={@attrs.step}
+      autocomplete="off"
+    />
+    """
+  end
+
   defp input_output(%{attrs: %{type: type}} = assigns)
        when type in [:number, :color, :url, :text] do
     ~H"""
     <input
-      type={html_input_type(@attrs.type)}
+      type={html_input_type(type)}
```

Comment:
We should only point to assigns, not local variables :)

```suggestion
      type={html_input_type(@attrs.type)}
```